README.md

```diff
@@ -2,68 +2,77 @@
 
 
 ## Overview
-OpenFin FDC3 provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
+
+The OpenFin FDC3 Service provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
 
 This project consist of 3 parts:
-1. The FDC3 Service, taking care of intents, context and resolving them (UI)
-2. The FDC3 Client, exposing APIs for applications to handle/raise intents with contexts
+1. The FDC3 Provider, taking care of contexts and intents, including the intent resolution UI
```

Comment:
"taking care of" feels a bit vague and informal. Can we do something like "The FDC3 Provider, handling contexts, intents, and the intent resolution UI."

Reply:
✅